components/brave_wallet/browser/swap_service.cc

```diff
@@ -105,8 +105,12 @@ GURL AppendJupiterQuoteParams(
     url = net::AppendQueryParameter(url, "outputMint", params.output_mint);
   if (!params.amount.empty())
     url = net::AppendQueryParameter(url, "amount", params.amount);
-  url = net::AppendQueryParameter(url, "feeBps",
-                                  brave_wallet::SwapService::GetFee(chain_id));
+
+  if (brave_wallet::HasJupiterFeesForTokenMint(params.output_mint)) {
+    url = net::AppendQueryParameter(
```

Comment:
Just noticed that for the URLs we're building in this file, we do not check if it's actually a valid URL before running `api_request_helper_.Request`, we should probably check if they're valid before it.
At least we should do it for GetPriceQuoteURL, GetTransactionPayloadURL, and GetJupiterQuoteURL which is not simply just replacing known URL with a pre-defined string constants, so it could be invalid in the end.

Reply:
Makes sense. I prefer to do this in a new PR, since this one is going to be uplifted to release, so want to keep the changeset small.

Created this issue as a follow-up: https://github.com/brave/brave-browser/issues/25721